server/src/main/java/timely/Server.java

```diff
@@ -142,25 +139,15 @@ private static boolean useEpoll() {
         }
     }
 
-    private static String usage() {
-
-        return "Server <configFile>";
-    }
-
     public static void fatal(String msg, Throwable t) {
         LOG.error(msg, t);
         LATCH.countDown();
     }
 
     public static void main(String[] args) throws Exception {
 
-        if (args.length != 1) {
-            System.err.println(usage());
-        }
-        final File conf = new File(args[0]);
-        if (!conf.canRead()) {
-            throw new RuntimeException("Configuration file does not exist or cannot be read");
-        }
+        Configuration conf = initializeConfiguration(args);
+
         Server s = new Server(conf);
```

Comment:
Ok I'll submit another ticket for the Server change. Sounds like it wouldn't be tightly coupled to this change.